packages/turf-polygon-smooth/index.js

```diff
@@ -18,7 +18,7 @@ import { featureCollection, polygon, multiPolygon } from "@turf/helpers";
  * //addToMap
  * var addToMap = [smoothed, polygon];
  */
-function polygonSmooth(inputPolys, options) {
+function polygonSmooth(inputPolys, options = {}) {
```

Comment:
@geirsagberg this is using an ES6 syntax and we currently only support es5 for `.js` files.

Check out [this handling](https://github.com/Turfjs/turf/blob/master/packages/turf-sector/index.js#L33-L34
) for how we've typically done this.

Reply:
Gotcha, fixed.